platform/windows/detect.py

```diff
@@ -180,37 +180,37 @@ def configure_msvc(env, manual_msvc_config):
 
     if env["target"] == "release":
         if env["optimize"] == "speed":  # optimize for speed (default)
-            env.Append(CCFLAGS=["/O2"])
+            env.Append(CCFLAGS=["/O2", "/MP"])
```

Comment:
Factor it out, you apply it in all branches.

Reply:
on it